### PR TITLE
Fix endless loop when the issue comments are paginated

### PR DESCRIPTION
**Pull request by milch**
See e.g. fastlane/fastlane#8431

There is no way to test this I think, but I wrote this method in `pry` and it seems to work: 

screen shot 2017-11-20 at 13 39 45

### CODE REVIEW

**Comment by milch** [comment]
Many thanks 💯 Don't have merge permissions though

File: bot.rb [hunk 1]
```diff
@@ -337,6 +337,13 @@ def process_code_signing(issue)
       return body.join("\n\n")
     end
 
+    def last_responding_user(issue)
+      client.issue_comments(SLUG, issue.number)
```
Comment (revolter) [inline bot.rb]: The return value of this function call isn't used or am I missing something?

Comment (milch) [reply bot.rb]: Yeah the GH API is a bit weird, you have to make the call so the API call is made. This returns the model objects, so to get the actual response object you have to make a separate call which I do in the next line:

```ruby
link_to_last_page = client.last_response.rels[:last]
```

Comment (revolter) [reply bot.rb]: Oh, I see, thank you!

**Pull request merged**